Classes/Service/FileGenerator.php

```diff
@@ -771,7 +771,7 @@ protected function generateDocumentationFiles()
     protected function generateEmptyGitRepository()
     {
         $targetDirectory = $this->extensionDirectory . '.git/';
-        if (is_file($targetDirectory) || is_dir($targetDirectory) || is_link($targetDirectory)) {
+        if (is_file($this->extensionDirectory . '.git') || is_dir($targetDirectory) || is_link($targetDirectory)) {
```

Comment:
how about removing the slash in line 773 and keep the line 774?

Reply:
You are right. I've changed it in Line 773 and restored 774.